### PR TITLE
test: Implement Outbound unittests with 100% coverage

### PR DESCRIPTION
**Pull request by victorpoluceno**

### CODE REVIEW

File: tests/helpers.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import asyncio
+from unittest import mock
+
+
+def make_coro_mock():
```
Comment (mauricioabreu) [inline tests/helpers.py:5]: The right way to do this is creating a "fixture" because the tests would receive this coroutine as an explicit parameter.

Comment (mauricioabreu) [inline tests/helpers.py:5]: I tried to make it work but for some reason it did not go well.

Comment (victorpoluceno) [reply tests/helpers.py:5]: You mean, the right way to provide that function to all tests?

Comment (mauricioabreu) [reply tests/helpers.py:5]: Yes! Dependency injection.

Comment (victorpoluceno) [reply tests/helpers.py:5]: @mauricioabreu I'm no sure about how to use fixtures on this case. Actually, the values of fixtures tha are inject, no the fixture itself. So, no sure if fixtures can help us here. 

I think the right thing to do would be to inject the object already mocked, but that will need one fixture per mock, not sure if it's a good idea also.

**Comment by victorpoluceno** [comment]
BTW, gonna merge this, but feel free to improve this code if you want to.

**Pull request merged**